sources/packages/api/src/route-controllers/program-year/program-year.controller.ts

```diff
@@ -0,0 +1,25 @@
+import { Controller, Get, NotFoundException } from "@nestjs/common";
+import { ProgramYearService } from "../../services";
+import { ProgramYearDto } from "./models/program-year.dto";
+import BaseController from "../BaseController";
+@Controller("program-year")
```

Comment:
Please add the proper authorization. It could be only students for now and we can add institutions later when they start consuming it.